docs/pipelines/add-pipeline.md

```diff
@@ -16,11 +16,13 @@ required_permissions:
 1. On Devopness, navigate to a project then select an environment
 1. Find the `Applications` card
 1. Click `View` in the `Applications` card to see a list of existing `Applications`
-1. In the list of applications find the application where you want to add a pipeline and click `DETAILS`
-1. On the upper-right corner of the applications details view click `SETTINGS`
-1. Use the drop-down menu to choose `PIPELINES`
-1. On the right corner of the list click `ADD PIPELINE`
-1. Follow the prompts to complete the configuration of the new `Pipeline`
+1. In the list of applications find the application where you want to add a pipeline and click in the `NAME` of the application
+1. On the left corner of the list click in `PIPELINES`
+1. On the right corner of the list click `MANAGE`
+1. Provide a `name` for the new pipeline
+1. Select the `OPERATION`
+    > Don't use the `CUSTOM` operation, it is not fully supported yet
+1. Provide the `max parallel instances` for the pipeline
```

Comment:
```suggestion
1. Provide a value for the `Max parallel actions` field
```

Reply:
I copied your suggestions into my own commit, because when you made them, I had already made another commit correcting some of the spelling errors, and then the interface did not release the 'commit' of your suggestions.